source/ControllableHero.hx

```diff
@@ -0,0 +1,54 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+
+using flixel.util.FlxSpriteUtil;
+
+// Part 2: Added a ControllableHero Class
+class ControllableHero extends FlxSprite
+{
+	// Movement variable
+	var speed = 10; // Couldn't figure out a better way to handle this?
```

Comment:
Creating your own variable to track the speed this object should move at is totally reasonable. Might be good to specify the type (int or float) to be absolutely clear. You can also add [access modifiers](https://haxe.org/manual/class-field-property.html) if you want to make this value totally unchangeable if you expect it to be a constant.